applications/GSoC/AR-weekly-updates/week5.md

```diff
@@ -0,0 +1,17 @@
+# Week 5 (5th July - 11th July)
+
+**Done**
+
+* PRs #114 and #115 got merged.
```

Comment:
Can you make these links to the PRs @anoura12 ?

Reply:
Oh yes sure! I forgot to do that here.